src/core/task1.js

```diff
@@ -1,4 +1,4 @@
-export function multiply() {
+export default function multiply() {
```

Comment:
What will be returned if we don't pass any arguments? The function should return 0 in this case.

Reply:
Nope, arguments always have a length property. You should check actually the count of arguments.